### DISCUSSION TITLE
unable to deploy on my custom domain gives CNAME error

### DISCUSSION THREAD

**Discussion by xkeshav** [community/community#81810] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have publicrepo which have workflow ( github action ) added. build is successful but deployement failed, which eventually succeed.

note that it has been verified using github settings . but DNS check process failed and give below error

```
Both www.<domain>.com and its alternate name are improperly configured
Your site's DNS settings are using a custom subdomain, www.<domain>.com, that is not set up with a correct CNAME record. We recommend you change this to a CNAME record pointing to xkeshav.github.io. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/)
```

if I remove www from custom domain and after DNC check process it gives diffrent error

```sh
Both domain.com and its alternate name are improperly configured
Domain does not resolve to the GitHub Pages server. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (NotServedByPagesError).

```

what is the issue here?  Do I wait for 24 hours to activate or this is issue because this domain is purchased from google domain and a website page is running on that domain.

**Comment by xkeshav** [comment]
someone care to reply?

**Reply by GovindPotdar** [reply]
Did you solved this issue, I think I am getting the same error

www.example.org is improperly configured
Your site's DNS settings are using a custom subdomain, www.example.org , that is set up as an A record. We recommend you change this to a CNAME record pointing to example.github.io. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidARecordError).
